project_zoo/animal.py

```diff
@@ -0,0 +1,29 @@
+# Calvin Hsieh chgenedu@gmail.com
+# oa-professional-software-development-bootcamp
+# Week 2
+# Nov 5, 2023
+#
+# animal.py
+
+from abc import abstractmethod
+
+class Animal:
+	def __init__(self, name, age):
+		self.__name = name
+		self.__age = age
```

Comment:
nit: usually its a good idea to leave one empty line between method definitions. Helps with readability, and most lint checkers would complain about this.

I would recommend setting up any sort of auto-formatter. I use ruff, or black

1. https://github.com/astral-sh/ruff
2. https://github.com/psf/black

You can probably find good VS Code formatter, setting that up might be easier.

Reply:
I have also provided a Github Action in the main fork, that will enable running the `ruff` formatter at every PR, so you can automatically check without setting anything up on locally. Let me know if you need help with that

Reply:
Thanks for your feedback. I have installed the ruff extension for vscode, but it would be helpful if you can give us a demo.